sdk/scheduler/src/main/java/com/mesosphere/sdk/curator/CuratorPersister.java

```diff
@@ -29,7 +29,7 @@
  */
 public class CuratorPersister implements Persister {
 
-    private static final Logger logger = LoggerFactory.getLogger(CuratorPersister.class);
+    private static final Logger logger = LoggingUtils.getLogger(CuratorPersister.class);
```

Comment:
A bit of a consistency nit. Should we use `LOGGER` or `logger` everywhere?

Reply:
In theory any `public static final` should be `LOGGER` while just `public final` should be `logger`. It looks like there are a few cases of `public static final logger`, so I'll update those. Didn't see any cases of the opposite.

Reply:
sorry missed the differences. Must have compared two inconsistent ones.